public/controllers/agent/components/register-agent.js

```diff
@@ -568,6 +571,7 @@ export class RegisterAgent extends Component {
               children: (
                 <EuiButtonGroup
                   color="primary"
+                  legend="Choose the architecture"
```

Comment:
whats the difference between this "Chose the architecture" and the previous one on line 557?

Reply:
Both legends represent the component `title`. It's for accessibility purposes. As you can see, both titles have the same text and so do their legends.